### DISCUSSION TITLE
Can one use the clickhouse client that's installed using curl with docker-version of clickhouse?

### DISCUSSION THREAD

**Discussion by aaronvg** [ClickHouse/ClickHouse#52548] [Q&A]
Anyone ever got this working? Or do I need to use `docker exec -it some-clickhouse-server clickhouse-client`?

Could you explain why if that is the case?

**Comment by tsolodov** [comment]
You can find working examples here: https://hub.docker.com/r/clickhouse/clickhouse-server/
